### ISSUE TITLE
infinite loop in nextChange around DST boundary (Safari)

### ISSUE THREAD

**Issue by atrick-speedline** [opening-hours/opening_hours.js#325]
As far as I know, this only affects safari, you can reproduce it with the following (assuming you're in the PST/PDT timezone region):

```js
const hours = new openingHours("01:00-02:00");
const start = moment("2020-03-02");
const end = moment(start).add(7, "d");
const values = hours.getOpenIntervals(start.toDate(), end.toDate()); // infinite loop here
console.info(values)
```

**Comment by ypid** [comment]
Hi @atrick-speedline 

Thanks for reporting.
I am not keen on trying to install Safari on Linux or setting up a OSX VM and then debug an issue that seems to only occur in a propitiatory browser. If you care, feel free to submit a patch, otherwise, I am afraid, I have not the time to look into that.

**Comment by atrick-speedline** [comment]
Yeah, that's pretty understandable. It's really hard for us to debug this as well. Hopefully someone with a copy of OSX comes along and sees this.
